statsd/client.py

```diff
@@ -197,6 +197,23 @@ def reconnect(self, data):
         self.connect()
 
 
+class UnixSocketStatsClient(TCPStatsClient):
```

Comment:
This really shouldn't extend `TCPStatsClient`, there's no "is a" relationship here. It should extend the base client directly